packages/console/src/components/Card/index.module.scss

```diff
@@ -4,4 +4,5 @@
   background: var(--color-card-background);
   border-radius: _.unit(4);
   padding: _.unit(6);
+  height: 100%;
```

Comment:
How would this component be used? Would `height: 100%` serve our purpose? As in most cases, setting height to 100% would not work as expected, right?

Reply:
i believe `height: 100%` serves the most situations of the `<Card>` component: page content, modal content, etc.